packages/builder/src/stores/builder/components.ts

```diff
@@ -1,3 +1,5 @@
+// TODO: analise and fix all the undefined ! and ?
```

Comment:
🥚 Just checking if this was intentionally left in

Reply:
Yes, this was left intentionally as it will require a much bigger refactor